### ISSUE TITLE
Add automatic 3-business-days expiration date (just visually)

### ISSUE THREAD

**Issue by Anbranin** [umts/st-pax#117]
Once a passenger registers, the default expiration date for rides should be set to "3 days from registration." But this need only be visual on the index/edit/show--the expiration date itself is a property of a doctor's note, and we shouldn't add the property of something that's not there.
A method on `Passenger` that returns an expiration date and sets it to either 3 days from registration or the expiration date of the doctor's note should suffice.
Although, I think setting "expiration_date" as a method on an object meant to represent a living being is a little terrible, so I'm open to alternatives.

**Comment by werebus** [comment]
I _will_ mention that "business days" was not implemented in #134, just "days".

**Comment by Anbranin** [comment]
That's true. I forgot that this was business-days because there's a separate issue specifically about business days. Since that issue exists I'll consider this OK but it was an oversight.
